pkg/cloud/azure/azure_storage.go

```diff
@@ -39,20 +39,30 @@ const (
 	AzureAccountKeyParam = "AZURE_ACCOUNT_KEY"
 	// AzureEnvironmentKeyParam is the query parameter for the environment name in an azure URI.
 	AzureEnvironmentKeyParam = "AZURE_ENVIRONMENT"
+
+	scheme = "azure"
```

Comment:
Do we want to differentiate between azure-kms and azure-storage?

Reply:
We could, I'm defaulting to the existing schemes that we support if there are no conflicts but we could start moving towards `azure-storage` and `azure-kms`. I'll make the change.

Reply:
done!